composer.json

```
@@ -65,7 +65,7 @@
         "magento/composer": "1.6.0",
         "magento/magento-composer-installer": ">=0.1.11",
         "magento/zendframework1": "~1.14.2",
-        "monolog/monolog": "^1.17",
+        "monolog/monolog": "~1.26.1",
```

Let's update monolog to the latest available version
```suggestion
        "monolog/monolog": "^2.2",
```